### PR TITLE
profile: add a document presenting Bump on the Github “bump-sh” profile

### PR DESCRIPTION
**Pull request by paulRbr**
Github allows to customize the public organisation
profile page (https://github.com/bump-sh) by defining a
profile/README.md file in the community repo.

Let's use it!

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+Share the link of your documentation to your API consumers and help them to never miss an API change again. Indeed, they will have the possibility to **track your API structural changes** if they subscribe by email. Once a week, if your API changed, subscribers will receive a changelog digest of your documentation. Curious? Here is the API [documentation changelog](https://developers.bump.sh/changes) of our own API.
+
+To make the most out of Bump, we suggest to integrate our tool in your development process. We offer a [Command Line Interface](https://github.com/bump-sh/cli/blob/master/package.json)) and [Continuous Integration](https://help.bump.sh/continuous-integration) examples to let you: view API diff during code review thanks to our [Github Action](https://github.com/bump-sh/github-action), Bump will automatically **comment your pull requests with changes digest**, then once merged we will **deploy your changes** directly to your documentation page.
```
Comment (thimy) [inline profile/README.md]:
```suggestion
To make the most out of Bump, we suggest integrating our tool in your development process. We offer a [Command Line Interface](https://github.com/bump-sh/cli/blob/master/package.json)) and a [Continuous Integration](https://help.bump.sh/continuous-integration) with examples. You will be able to view API diff during code review thanks to our [Github Action](https://github.com/bump-sh/github-action). Bump will automatically **comment your pull requests with changes digest** and **deploy your changes** directly to your documentation page once they are merged.
```

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  <a href="https://bump.sh/users/sign_up">Sign up</a>
+</p>
+
+[![Bump.sh](https://img.shields.io/badge/Bump.sh-https://bump.sh-025fd7.svg)](https://bump.sh)
+[![Twitter](https://img.shields.io/badge/Twitter-@bump_hq-9cf.svg)](https://twitter.com/bump_hq)
+[![LinkedIN](https://img.shields.io/badge/LinkedIn-@bump_sh-lightgray.svg)](https://www.linkedin.com/company/bump-sh)
+
+Bump allows you to easily **generate your API documentation and API changelog**.
+
+From an [OpenAPI](https://spec.openapis.org/oas/latest.html) or [AsyncAPI](https://www.asyncapi.com/) specification, you can deploy your **API documentation in a couple of minutes**. Some UI customization are available (title, colors, menu), you can protect your documentation page with a password, customize the URL with your own domain.
```
Comment (thimy) [inline profile/README.md]:
```suggestion
From an [OpenAPI](https://spec.openapis.org/oas/latest.html) or [AsyncAPI](https://www.asyncapi.com/) specification, you can deploy your **API documentation in a couple of minutes**. You can customize your documentation UI (title, colors, menu), protect it with a password, and customize its URL with your own domain.
```

File: profile/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+Share the link of your documentation to your API consumers and help them to never miss an API change again. Indeed, they will have the possibility to **track your API structural changes** if they subscribe by email. Once a week, if your API changed, subscribers will receive a changelog digest of your documentation. Curious? Here is the API [documentation changelog](https://developers.bump.sh/changes) of our own API.
```
Comment (thimy) [inline profile/README.md]:
```suggestion
Share a link to your documentation with your API consumers and help them never miss an API change again. They will have the possibility to **track API structural changes** if they subscribe by email. Once a week, subscribers will receive a changelog digest of your documentation if your API has changed. Curious? Here is the API [documentation changelog](https://developers.bump.sh/changes) of our own API.
```

File: profile/README.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+To make the most out of Bump, we suggest to integrate our tool in your development process. We offer a [Command Line Interface](https://github.com/bump-sh/cli/blob/master/package.json)) and [Continuous Integration](https://help.bump.sh/continuous-integration) examples to let you: view API diff during code review thanks to our [Github Action](https://github.com/bump-sh/github-action), Bump will automatically **comment your pull requests with changes digest**, then once merged we will **deploy your changes** directly to your documentation page.
+
+If you work in a multiple services environment and have many different APIs, we got your covered. You can organize, mix your OpenAPI and AsyncAPI based documentations, in a developer Hub. You will thus have **all of your APIs listed in one place**. Check our [demo “Train company” hub](https://demo.bump.sh/) for a live example.
```
Comment (thimy) [inline profile/README.md]:
```suggestion
If you work in a multiple services environment and have many different APIs, we got you covered. You can organize and mix OpenAPI and AsyncAPI based documentations in a developer Hub. You will thus have **all of your APIs listed in one place**. Check our [demo “Train company” hub](https://demo.bump.sh/) for a live example.
```

**Review by thimy — COMMENTED** [review]
I suggested a few improvements on phrasing :)

**Comment by paulRbr** [comment]
Thanks @thimy for the review (for info it's the text we have on https://help.bump.sh/ so I'll update that too)

**Review by antlio — APPROVED** [review]
that's great thanks!

I think at some point it should be sync with our homepage value prop (cc @gregbump).

**Pull request merged**